docs/docs/icicle/overview.md

```diff
@@ -5,8 +5,14 @@ title: ICICLE Overview
 
 # ICICLE Overview
 
+<p align="center">
+  <img src="https://github.com/user-attachments/assets/794c306a-b5fc-425c-8b1b-60b87aa12200" alt="Untitled design (29)" width="500"/>
+</p>
+
+
 ## What is ICICLE?
 
+
```

Comment:
No need for another blank line here

Reply:
theres a static bade - unsure what u mean